smart-contracts/assembly/contracts/FT/token-commons.ts

```diff
@@ -47,27 +48,35 @@ function getBalanceKey(address: Address): StaticArray<u8> {
  * Sets the allowance of the spender on the owner's account.
  *
  * @param owner - owner address
- * @param spenderAddress - spender address
+ * @param spender - spender address
  * @param amount - amount to set an allowance for
  */
-export function _approve(
-  owner: Address,
-  spenderAddress: Address,
-  amount: u256,
-): void {
-  const key = stringToBytes(owner.toString().concat(spenderAddress.toString()));
+export function _approve(owner: Address, spender: Address, amount: u256): void {
+  const key = getAllowanceKey(owner, spender);
+
   Storage.set(key, u256ToBytes(amount));
 }
 
 /**
  * Returns the allowance set on the owner's account for the spender.
  *
  * @param owner - owner's id
- * @param spenderAddress - spender's id
+ * @param spender - spender's id
  *
  * @returns the allowance
  */
-export function _allowance(owner: Address, spenderAddress: Address): u256 {
-  const key = stringToBytes(owner.toString().concat(spenderAddress.toString()));
+export function _allowance(owner: Address, spender: Address): u256 {
+  const key = getAllowanceKey(owner, spender);
   return Storage.has(key) ? bytesToU256(Storage.get(key)) : u256.Zero;
 }
+
+/**
+ * @param owner - address of the token owner
+ * @param spender - address of the token spender
+ * @returns the key of the allowance in the storage for the given addresses
+ */
+function getAllowanceKey(owner: Address, spender: Address): StaticArray<u8> {
```

Comment:
As your function is not "getting" the data remotley and allowanceKey is not an attribute, you should remove the useless get prefix.

```suggestion
function allowanceKey(owner: Address, spender: Address): StaticArray<u8> {
```